From d1b0a6d05ab083af516f038365a8ffcac0f50d02 Mon Sep 17 00:00:00 2001
From: Tomas Henzl <thenzl@redhat.com>
Date: Thu, 27 Nov 2014 16:33:33 -0500
Subject: [scsi] megaraid_sas: driver version upgrade and remove some meta data of driver (06.805.06.01-rc1)

Message-id: <1417106019-4286-2-git-send-email-thenzl@redhat.com>
Patchwork-id: 100993
O-Subject: [PATCH 1/7] megaraid_sas: driver version upgrade and remove some meta data of driver (06.805.06.01-rc1)
Bugzilla: 1162645
RH-Acked-by: Maurizio Lombardi <mlombard@redhat.com>
RH-Acked-by: Rob Evers <revers@redhat.com>

Update driver version and remove some meta data (release date and extended
version) about megaraid_sas driver.

Signed-off-by: Sumit Saxena <sumit.saxena@avagotech.com>
Reviewed-by: Tomas Henzl <thenzl@redhat.com>
Signed-off-by: Christoph Hellwig <hch@lst.de>
(cherry picked from commit d98a6deb75bce12784c0c7f1fdc02e6a67adac4d)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/megaraid/megaraid_sas.h b/drivers/scsi/megaraid/megaraid_sas.h
index a49914d..fe546e6 100644
--- a/drivers/scsi/megaraid/megaraid_sas.h
+++ b/drivers/scsi/megaraid/megaraid_sas.h
@@ -33,9 +33,7 @@
 /*
  * MegaRAID SAS Driver meta data
  */
-#define MEGASAS_VERSION    "06.805.06.00-rc1"
-#define MEGASAS_RELDATE    "Sep. 4, 2014"
-#define MEGASAS_EXT_VERSION   "Thu. Sep. 4 17:00:00 PDT 2014"
+#define MEGASAS_VERSION    "06.805.06.01-rc1"
 
 /*
  * Device IDs
diff --git a/drivers/scsi/megaraid/megaraid_sas_base.c b/drivers/scsi/megaraid/megaraid_sas_base.c
index 4e6768d..1aa6972 100644
--- a/drivers/scsi/megaraid/megaraid_sas_base.c
+++ b/drivers/scsi/megaraid/megaraid_sas_base.c
@@ -6413,16 +6413,6 @@ static ssize_t megasas_sysfs_show_version(struct device_driver *dd, char *buf)
 static DRIVER_ATTR(version, S_IRUGO, megasas_sysfs_show_version, NULL);
 
 static ssize_t
-megasas_sysfs_show_release_date(struct device_driver *dd, char *buf)
-{
- return snprintf(buf, strlen(MEGASAS_RELDATE) + 2, "%s\n",
-   MEGASAS_RELDATE);
-}
-
-static DRIVER_ATTR(release_date, S_IRUGO, megasas_sysfs_show_release_date,
-     NULL);
-
-static ssize_t
 megasas_sysfs_show_support_poll_for_event(struct device_driver *dd, char *buf)
 {
  return sprintf(buf, "%u\n", support_poll_for_event);
@@ -6725,8 +6715,7 @@ static int __init megasas_init(void)
  /*
   * Announce driver version and other information
   */
- printk(KERN_INFO "megasas: %s %s\n", MEGASAS_VERSION,
-        MEGASAS_EXT_VERSION);
+ pr_info("megasas: %s\n", MEGASAS_VERSION);
 
  spin_lock_init(&poll_aen_lock);
 
@@ -6761,10 +6750,6 @@ static int __init megasas_init(void)
       &driver_attr_version);
  if (rval)
   goto err_dcf_attr_ver;
- rval = driver_create_file(&megasas_pci_driver.driver,
-      &driver_attr_release_date);
- if (rval)
-  goto err_dcf_rel_date;
 
  rval = driver_create_file(&megasas_pci_driver.driver,
     &driver_attr_support_poll_for_event);
@@ -6788,12 +6773,7 @@ err_dcf_support_device_change:
 err_dcf_dbg_lvl:
  driver_remove_file(&megasas_pci_driver.driver,
    &driver_attr_support_poll_for_event);
-
 err_dcf_support_poll_for_event:
- driver_remove_file(&megasas_pci_driver.driver,
-      &driver_attr_release_date);
-
-err_dcf_rel_date:
  driver_remove_file(&megasas_pci_driver.driver, &driver_attr_version);
 err_dcf_attr_ver:
  pci_unregister_driver(&megasas_pci_driver);
@@ -6813,8 +6793,6 @@ static void __exit megasas_exit(void)
    &driver_attr_support_poll_for_event);
  driver_remove_file(&megasas_pci_driver.driver,
    &driver_attr_support_device_change);
- driver_remove_file(&megasas_pci_driver.driver,
-      &driver_attr_release_date);
  driver_remove_file(&megasas_pci_driver.driver, &driver_attr_version);
 
  pci_unregister_driver(&megasas_pci_driver);
-- 
1.7.1